docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1464,10 +1465,18 @@ be selected directly, or used in conditional statements. For example, you can
 inspect the file path for each record:
 
 ```sql
-SELECT *, "$path", "$file_modified_time"
+SELECT *, "$partition", "$path", "$file_modified_time"
 FROM example.web.page_views;
 ```
 
+Retrieve all records that belong to a specific partition using `"$partition"` filter:
+
+```sql
+SELECT *
+FROM example.web.page_views
+WHERE "$partition" = 'part=2022-07-01'
```

Comment:
This feels rather unintuitive from UX perspective.
Why would one user need to add `part=...` in the filter?

Also as mentioned in another comment, we need to think at composed partition specs and other transformations.

I'm having a hard time to imagine that an end-user would/should know about `part_bucket` .

Reply:
Removed filter example. I think projection example is sufficient.

We have already exposed the internal partition name (`part_bucket`) via the `$partitions` table. This is not for end-users like analysts. This is for administrators.